### PR TITLE
Fixed the id generating issue in Education array

### PR DESCRIPTION
**Pull request by tanujvyas10**
Fixes Issue no. #71
@jaskirat2000 @devesh-verma Please review

### CODE REVIEW

**Comment by tanujvyas10** [comment]
@jaskirat2000 @devesh-verma @vaibhavdaren Please review my PR

File: app/models/User.js [hunk 1]
```diff
@@ -114,6 +114,7 @@ const UserSchema = new mongoose.Schema({
       },
       education: [
         {
```
Comment (vaibhavdaren) [inline app/models/User.js]: all backend pr need to be with atleast 2  test cases 1 pass 1 fail

Comment (tanujvyas10) [reply app/models/User.js]: can you please be more specific , what you mean?

**Comment by devesh-verma** [comment]
@tanujvyas10 hey this is addressed by @Rupeshiya in #72

**Comment by tanujvyas10** [comment]
@devesh-verma , my PR was specifically for UserSchema
@Rupeshiya  PR was specified for schema for projects
I could get where this issue has been address in #72
Please clarify this

**Comment by devesh-verma** [comment]
@tanujvyas10 my bad it is in #76

**Comment by tanujvyas10** [comment]
@devesh-verma its fine sir

**Pull request closed without merging**